wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1221,12 +1259,14 @@ instance ToSchema WalletAddress where
             & ("id"            --^ "Actual address.")
             & ("used"          --^ "True if this address has been used.")
             & ("changeAddress" --^ "True if this address stores change from a previous transaction.")
+            & ("isOurs"        --^ "'is-ours' if this address is recognised as ours, 'not-recognised' if it has not (yet) been encountered")
```

Comment:
I was thinking (perhaps @KtorZ can give his own cents): when we use the `is` suffix usually that is used for booleans, but this one is not. Should we have rather a `ownership` field? I think it sounds much better.

Reply:
Agreed. `is` or `has` prefix usually implies a boolean kind of value which is anyway a bad thing to have in an API :stuck_out_tongue:  
I like `ownership` but I'd rather have the haskell data-type match as well and renamed to `AddressOwnership` if we go for this.